WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -223,4 +285,29 @@ public function process_matched_token( $stackPtr, $group_name, $matched_content
 		}
 	}
 
+	/**
+	 * Determine based on the "raw" parameter value, whether a file parameter points to
+	 * a local data stream.
+	 *
+	 * @param string $raw_param_value Raw parameter value.
+	 *
+	 * @return bool True if this is a local data stream. False otherwise.
+	 */
+	protected function is_local_data_stream( $raw_param_value ) {
```

Comment:
It's possible to create [custom stream wrappers](https://secure.php.net/manual/en/function.stream-wrapper-register.php) (we have one for VIP: `vip://`), so I'd like to see a public property that would allow for this to be added at the ruleset level, and then added as part of a check here.

Reply:
I think those kind of wrappers would be extremely rare in the context of WP and depending on how they can be used shouldn't necessarily be whitelisted - similar to the PHP native `php://filter` wrapper which I didn't include in this sniff on purpose.

As for VIPCS, this sniff could just be extended by a VIP native sniff and the extra wrapper added to the existing properties in a `__construct()` method.